### PR TITLE
CMT-6980 - DBaaS - MongoDB - Deploy a DB using Terraform within a vRack

### PR DESCRIPTION
**Pull request by OVH-Team-Guides**

### CODE REVIEW

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md [hunk 1]
```diff
@@ -0,0 +1,303 @@
+---
+title: MongoDB - Tutorial - How to deploy a MongoDB database on OVHcloud using Terraform
+excerpt: Find out how to to set up an OVHcloud managed database service via Terraform within a vRack, as well as how to access it from other parts of your infrastructure.
```
Comment (thild42) [inline pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md]: an OVHcloud managed MongoDB service

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+updated: 2024-01-29
+---
+
+## Objective
+
+The vRack offered by OVHcloud is a private network solution that allows our customers to route their traffic between their OVHcloud services, eg. instances, physical dedicated servers, or, in this case, a managed database service.
+
+**This tutorial shows you how to set up an OVHcloud managed database service via Terraform within a vRack, as well as how to access it from other parts of your infrastructure.**
```
Comment (thild42) [inline pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md]: an OVHcloud managed MongoDB service

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md [hunk 3]
```diff
@@ -0,0 +1,303 @@
...
+By default, the cluster will be delivered with the user `admin@admin`, whose password you can reset via the OVHcloud Control Panel to log in directly.
+
+You can also add another user via Terraform, with full wide read/write permissions on all databases, as follows:
+
+```bash
+resource "ovh_cloud_project_database_mongodb_user" "tf_user" {
+  service_name = ovh_cloud_project_database.service.service_name
+  cluster_id   = ovh_cloud_project_database.service.id
+  name         = "tf_mongodb_user"
+  roles        = ["readWriteAnyDatabase"]
```
Comment (lpatte) [inline pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md]: It's work with 0.35.0 but after ovh provider 0.37.0, authentication will be mandatory, so readWriteAnyDatabase@admin

Comment (OVH-Team-Guides) [reply pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.en-gb.md]: Thank you for the review @lpatte . Can you please confirm the new version is now ok ?

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md [hunk 4]
```diff
@@ -0,0 +1,301 @@
...
+}
+
+resource "openstack_networking_subnet_v2" "my_subnet" {
+  network_id      = openstack_networking_network_v2.my_private_network.id
+  name            = "mySubnet"
+  region          = var.network.region
+  cidr            = "192.168.12.0/24"
+  enable_dhcp     = true
+  no_gateway      = false
+  dns_nameservers = ["1.1.1.1", "1.0.0.1"]
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md]: Update dns server to ovh 213.186.33.99

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md [hunk 5]
```diff
@@ -0,0 +1,301 @@
...
+Apply complete! Resources: 5 added, 0 changed, 0 destroyed.
+
+Outputs:
+
+cluster_uri = "mongodb+srv://<username>:<password>@mongodb-xxx-yyy.database.cloud.ovh.net/admin?replicaSet=replicaset&tls=true"
+user_name = "tf_mongodb_user@admin"
+user_password = <sensitive>
+```
+
+### Connection test using an instance of the same private network
```
Comment (yomovh) [inline pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md:260]: I would add an explanation on  how to access to a private instance through a floating IP with this page the https://help.ovhcloud.com/csm/en-gb-public-cloud-network-attach-floating-ip-to-instance?id=kb_article_view&sysparm_article=KB0050231

**Review by yomovh — COMMENTED** [review]
See on comment to take into account and one suggestion.

File: pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md (hunk 5 above)
Comment (OVH-Team-Guides) [reply pages/public_cloud/public_cloud_databases/mongodb_14_deploy_with_terraform/guide.de-de.md:260]: @yomovh , please let us know if https://github.com/ovh/docs/pull/5730/commits/2b1a7bb090fd78fce8b45cae29c87f75de4445de fits your suggestion

**Pull request merged**